torch/_inductor/codegen/triton.py

```diff
@@ -3880,18 +3881,53 @@ def __del__(self):
                 mutations=mutations,
                 index_dtype=index_dtype,
             )
-
             self.codegen_node_schedule_with_kernel(node_schedule, kernel)
             with config.patch("benchmark_kernel", True), V.set_kernel_handler(kernel):
                 src_code = kernel.codegen_kernel()
+
+            kernel_args = kernel.args
+            _, call_args, _ = kernel_args.python_argdefs()
+
         else:
             template_node = nodes[0]
             epilogue_nodes = nodes[1:]
 
-            with config.patch("benchmark_kernel", True):
-                src_code = self.codegen_template(
-                    template_node, epilogue_nodes, only_gen_src_code=True
+            # TODO - handle epilogue inputs as well
+            non_out_arg_indices = list(
```

Comment:
This part of code in unnecessary since you overwrite `non_out_arg_indices` below?

Reply:
Oh yea, this was from an old version. will delete